internal/cmd/connect/command_plugin_describe.go

```diff
@@ -19,7 +19,7 @@ type pluginDescribeOut struct {
 
 func (c *pluginCommand) newDescribeCommand() *cobra.Command {
 	cmd := &cobra.Command{
-		Use:               "describe <plugin>",
+		Use:               "describe <plugin-name>",
```

Comment:
Since this command has "plugin" in its name, this seems redundant.
```suggestion
		Use:               "describe <name>",
```

Reply:
I did it for the sake of consistency because that's how it's displayed in `pluginListOut`.

How about keep it as `plugin-name` here, and change both to `name` in v4 (because you're right that it's redundant)?

Reply:
This isn't a breaking change, so we change both to "name" now!

Reply:
That would make the human and seralized versions out of sync because
`Class string `human:"Plugin Name" serialized:"plugin_name"`
changing `plugin_name` to `name` would be a breaking change, right?

Reply:
I thought we were just changing the usage strings in the PR, not the struct fields!

Reply:
Right. What I mean is that I'm matching the argument label in the usage string to the corresponding field output in the struct for consistency.

But like broker, I'll change this to `name` in the usage string and make a v4 ticket to change redundant output labels in the structs.